### PR TITLE
Add brief explanation of private channel usage.

### PR DESCRIPTION
**Pull request by ayust**

### CODE REVIEW

File: lib/commands/list-private.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function formatChannel (chan, user) {
 
 function displayGroupList (filter, user, groups) {
   return {
-    text: `Available private channels: `,
+    text: `These channels are private to disallow previewing (view wihtout join). Available private channels: `,
```
Comment (varjmes) [inline lib/commands/list-private.js]: hey! little spelling mistake here,

`wihtout` should be `without` :)

**Review by aredridel — CHANGES_REQUESTED** [review]
I like the change, but it does have a typo!

File: lib/commands/list-private.js (hunk 1 above)
Comment (ayust) [reply lib/commands/list-private.js]: fixed

**Comment by varjmes** [comment]
Looks good to me! Thank you @ayust :)

**Pull request merged**